packages/grafana-ui/src/components/Button/styles.ts

```diff
@@ -133,8 +133,7 @@ export const getButtonStyles = stylesFactory(({ theme, size, variant, withIcon }
     `,
     icon: css`
       label: button-icon;
-      margin-right: ${iconDistance};
-      filter: brightness(100);
```

Comment:
Interestingly the white color was due to this. Not sure if it meant just to tweak the brightness a little bit or what. Right now the icon just gets the color of the text which seems to work ok.